ci/scripts/rust_example.sh

```diff
@@ -19,7 +19,6 @@
 
 set -ex
 cd datafusion-examples/examples/
-cargo fmt --all -- --check
```

Comment:
why remove this? I think it is intended to ensure that cargo fmt is run on the examples.

Reply:
Good question, and i see how this is not obvious. That's why it's separate commit with an explanation:

```
Remove doubled fmt check from CI

Examples' format is checked in `check-fmt` job, so can be skipped in
`rust_example.sh`.
```